releasetasks/templates/desktop/release_graph.yml.tmpl

```diff
@@ -126,6 +126,13 @@ tasks:
           {% include "checksums.yml.tmpl" %}
       {% endmacro %}
       {{ checksums_tasks()|indent(4) }}
+
+      {% if binary_transparency_enabled %}
```

Comment:
Might wanna rewrite this to `{% if binary_transparency_enabled is defined and binary_transparency_enabled %}`. I'll go into more details in the sum-up comment of the review.